schemas/definitions-find.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$id": "https://api.clearlydefined.io/schemas/definitions-find.json#",
```

Comment:
probably should look at the swagger as well. don't remember where we are on using the JSON schema in the swagger.